libs/libxx/libcxx.defs

```diff
@@ -20,22 +20,61 @@
 
 VERSION=11.0.0
 
-$(TOPDIR)/include/libcxx:
+libcxx-$(VERSION).src.tar.xz:
```

Comment:
why need keep the tarball?

Reply:
because
1. it's convenient for me during development
2. it can be useful for offline build